include/rms/rms.hh

```diff
@@ -13,25 +13,52 @@
 #include <cmath>
 #include <numeric>
 
+/** rms namespace
+ *  Contains the RMS class.
+ */
 namespace rms {
 
+/** RMS (Root Mean Square) Class.
+ *  This class determines the average power of a signal.
+ *  @tparam num_samples The number of samples to include in the ring buffer.
```

Comment:
When I first read this it wasn't quite clear to me what the module does.  I think it would be worth adding a note mentioning that the ring buffer will hold on to previous values if the buffer is bigger than the number of samples in the audio input. This would make it clear the module holds a history and can operate on more than one audio sample array. 

It may also be worth giving an example of sample rate and buffer size.

Reply:
![image](https://user-images.githubusercontent.com/10051310/232323378-56cd5432-ea74-48e0-ac52-e77e7f01d1d2.png)
How is this?

Reply:
updated

Reply:
That's a nice elegant way of writing that, I think that's better.